src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -140,163 +133,117 @@
+(defn routes-view
+  [current-screen-id]
+  (let [token-by-symbol                (rf/sub [:send-input-amount-screen/token-by-symbol])
+        token                          (rf/sub [:wallet/wallet-send-token])
+        routes                         (rf/sub [:send-input-amount-screen/routes])
+        valid-input?                   (rf/sub [:send-input-amount-screen/valid-input?])
+        amount-in-crypto               (rf/sub [:send-input-amount-screen/amount-in-crypto])
+        limit-exceeded?                (rf/sub [:send-input-amount-screen/upper-limit-exceeded?])
+        unsupported-token-in-receiver? (rf/sub
+                                        [:send-input-amount-screen/unsupported-token-in-receiver?])
+        request-fetch-routes           (fn [bounce-duration-ms]
+                                         (fetch-routes
+                                          {:amount                 amount-in-crypto
+                                           :valid-input?           valid-input?
+                                           :bounce-duration-ms     bounce-duration-ms
+                                           :token                  token
+                                           :reset-amounts-to-zero? (and limit-exceeded?
+                                                                        (some? routes))}))]
+    [routes/view
+     {:token                                     token-by-symbol
+      :send-amount-in-crypto                     amount-in-crypto
+      :valid-input?                              valid-input?
+      :token-not-supported-in-receiver-networks? unsupported-token-in-receiver?
+      :current-screen-id                         current-screen-id
+      :request-fetch-routes                      request-fetch-routes}]))
+
+(defn bottom-actions
+  [button-one-label button-one-props on-confirm]
+  (let [token-by-symbol                  (rf/sub [:send-input-amount-screen/token-by-symbol])
+        loading-routes?                  (rf/sub [:wallet/wallet-send-loading-suggested-routes?])
+        amount-in-crypto                 (rf/sub [:send-input-amount-screen/amount-in-crypto])
+        sending-to-unpreferred-networks? (rf/sub
+                                          [:send-input-amount-screen/sending-to-unpreferred-networks?])
+        current-color                    (rf/sub [:wallet/current-viewing-account-color])
+        not-enough-asset?                (rf/sub [:send-input-amount-screen/not-enough-asset?])
+        should-try-again?                (rf/sub [:send-input-amount-screen/should-try-again?])
+        confirm-disabled?                (rf/sub [:send-input-amount-screen/confirm-disabled?])]
+    [quo/bottom-actions
+     {:actions          :one-action
+      :button-one-label (if should-try-again?
+                          (i18n/label :t/try-again)
+                          button-one-label)
+      :button-one-props (merge (when-not should-try-again?
```

Comment:
`merge` is one of the slowest functions in Clojure, so it's good to avoid it. Here it looks like you can use `cond->` and `assoc`.